middleware/serverError/handler.go

```diff
@@ -54,11 +56,14 @@ func (rI *responseInterceptor) callRenderer(code int, title, description string)
 	if err != nil {
 		return err
 	}
+	preferencesCookie := cookies.GetCookiePreferences(rI.req)
 	data := map[string]interface{}{
 		"error": map[string]interface{}{
 			"title":       title,
 			"description": description,
 		},
+		"cookies_preferences_set": preferencesCookie.IsPreferenceSet,
```

Comment:
More of a question really: Does `IsPreferenceSet` return the default/zero bool value if it's not set? Just wondering if this could be nil and if so should we default to false?